content/en/docs/tasks/traffic-management/ingress/secure-ingress-sds/index.md

```diff
@@ -252,7 +252,7 @@ need to create secrets for multiple hosts and update the gateway definitions.
 
     {{< text bash >}}
     $ pushd mtls-go-example
-    $ ./generate.sh httpbin.example.com <password>
+    $ ./generate.sh httpbin.example.com password
```

Comment:
Shouldn't it instead be `<password>` on line 65 and in both places the instructions should tell the user to change `<password>` to some appropriate password? That's similar to how it's done in lots of other docs.

Reply:
@frankbu The results of `find . -name "*.md" -exec grep "./generate.sh" \{\} \; -print` are 4 examples with `<password>`, and 8 examples with `password`.

Shall I rework the PR to set them all to `<password>`?

Reply:
Yes, I think all this needs to be `<password>`